package.json

```diff
@@ -43,16 +43,17 @@
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-typescript": "^7.12.7",
     "@types/chrome": "^0.0.155",
-    "@types/jest": "^26.0.20",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.11.21",
```

Comment:
Based on the matrix of runtimes we define for the CI, shouldn't those types be downgraded to match Node 16?